lib/utils/file_download_util.dart

```diff
@@ -70,3 +71,26 @@ Uint8List getFileKey(ente.File file) {
       CollectionsService.instance.getCollectionKey(file.collectionID!);
   return CryptoUtil.decryptSync(encryptedKey, collectionKey, nonce);
 }
+
+Future<Uint8List> getFileKeyUsingBgWorker(ente.File file) async {
```

Comment:
getFileKey also maintains a in-memory cache (as fileKey doesn't change that). We should first check the cache and use the value if present.

Reply:
Seems like I was wrong. We don't maintain a cache for fileKeys. It's not super critical to have done.